TailLight/driver.cpp

```diff
@@ -28,13 +43,3 @@ NTSTATUS DriverEntry(
     return status;
 }
 
-
-/** Driver unload callback.
-    Used to perform operations that must take place before the driver is unloaded.  */
-VOID EvtDriverUnload(
```

Comment:
Why are you removing the `EvtDriverUnload` callback this way? Can't you just leave the function as-is?

Reply:
Please don't resolve this comment before it have been properly addressed.